src/main/java/bio/terra/workspace/service/resource/controlled/flight/create/CreateControlledResourceFlight.java

```diff
@@ -27,10 +27,7 @@
    * wait forever.
    */
   private static final RetryRule NOTEBOOK_GCP_RETRY_RULE =
-      new RetryRuleExponentialBackoff(
-          /* initialIntervalSeconds= */ 1,
-          /* maxIntervalSeconds= */ 8,
-          /* maxOperationTimeSeconds= */ 30);
+      new RetryRuleFixedInterval(/* intervalSeconds= */ 10, /* maxCount=  */ 10);
```

Comment:
RetryRule objects are stateful, I think you need a different object for each step here. Otherwise, you're sharing 10 total retries across both steps.

(We've definitely mixed this up elsewhere, PF-557 for other ones)

Reply:
I did not know that. For my 2c, PF-557 SG. I don't know when I'd want to share a retry rule, so changing the `addStep` api to take a factory for retry rules to prevent that from happening seems less error prone.

Reply:
whoops, Dan's right here. I didn't see that stairway re-initialized RetryRules for each step